sequencing/ATACSeq/README.md

```diff
@@ -0,0 +1,201 @@
+
+##### -scripts executed-
+- ATACSeq/preprocessing/progressReport.sh : identifies how many samples have been successful at each stage of the processing pipeline and for each fastq file, how far through the process it has progressed.
+- ATACSeq/preprocessing/countMTReads.sh : collates counts of the number of reads aligned to MT chromosome
+- ATACSeq/preprocessing/collateFlagStatOutput.sh : collates flagstat summary of aligned sorted reads.
+- ATACSeq/preprocessing/collateDataQualityStats.Rmd : generates Rmarkdown report summarising stage 1 qc metrics: raw reads metrics, trimming, alignment and peak calling
+
+##### -parameters-
+- `--array` : should be 0 for this script as general analysis is run rather than individual samples.
+- `(project-name)` project's directory.
+-optional-
+- `[STEPS]` They may be combined, with desired steps included as single string, i.e. MULIQC,COLLATE. Default if left blank is to run all of them.
+  - `MULTIQC`: Collates fastqc and alignment statistics in a single report.
+  - `COLLATE`: Collate results from all samples to show progress of each sample through the pipeline in order to avoid missing steps.
+  - `SUMMARY`: Collates stage 1 QC and peak calling results in a single Rmarkdown report.
+  - `BATCH`: If different batches of samples are found, results related to these are compared.
+
+
+### 5. Sex chromosomes 
+
+  `sbatch --array=<number of batch jobs> ATACSeq/jobSubmission/5_batchFormatSexChrs.sh (project directory) [STEPS]`
+  
+This script creates bam files for each sample containing only reads aligned to sex chromosomes, perform peak calling on these and output results.
+
+##### -scripts executed-
+- ATACSeq/preprocessing/subsetSexChrs.sh : subsets reads of only X and Y chromosomes for input sample in bam file. 
+	* requires a file in METADATA folder called passStage1SampleList.txt which lists the samples to be included for sex check
+- ATACSeq/preprocessing/sexChrPeaks.sh : performs peak calling on the sex chromomes, filter and read counts using MACS3 in Single-end mode.
+- ATACSeq/preprocessing/collateSexChecks.r : collates peak calling results on sex chromosomes and uses it to check the assigned sex of the sample.
+
+##### -parameters-
+- `--array` : number of batch jobs, each number matches a sample.
+- `(project-directory)` project's directory.
+-optional-
+- `[STEPS]` They may be combined, with desired steps included as single string, i.e. MULIQC,COLLATE. Default if left blank is to run all of them.
+  - `SPLIT`: Generate only samples with X and Y chromosomes reads.
+  - `PEAKS`: Perform only peak calling on new sex chromosomes reads samples.
+  - `CHECK`: Collate peak calling on sex chromosomes results and check assigned sex of each sample.
+
+### 6. Genotype check
+
+  `sbatch --array=<number of batch jobs> ATACSeq/jobSubmission/6_batchRunGenotypeConcordance.sh (project directory) [STEPS]`
```

Comment:
do we need to provide guidance on how many batch jobs need to be specified? If I recall it is the same as the number of samples?